.github/.workflows/golangci-lint.yml

```diff
@@ -0,0 +1,14 @@
+name: golangci-lint
+on:
+  pull_request:
+jobs:
+  golangci:
+    name: lint
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: golangci-lint
+        uses: golangci/golangci-lint-action@v2
+        with:
+          version: v1.29
```

Comment:
Let's go start with the latest `v1.31.0` release

https://github.com/golangci/golangci-lint/releases